python/graphscope/experimental/nx/tests/algorithms/builtin/test_ctx_builtin.py

```diff
@@ -142,6 +142,23 @@ def test_single_source_dijkstra_path_length(self):
         ans = dict(ret.values)
         assert replace_with_inf(ans) == self.p2p_length_ans
 
+        # test subgraph and edge_subgraph with p2p_undirected
```

Comment:
For these tests, I suggest we induce a REAL subgraph for verification.

Reply:
fix, the pr of https://github.com/GraphScope/gstest/pull/35 gstest need to merge first